src/transformers/utils/fx.py

```diff
@@ -59,6 +60,12 @@
 _IS_IN_DEBUG_MODE = os.environ.get("FX_DEBUG_MODE", "").upper() in ENV_VARS_TRUE_VALUES
 
 
+DEVICE = "meta"
```

Comment:
I would rather add an environment variable `FX_DEVICE_FOR_TRACING` with a default to `meta`, and set that to `cpu` for the tests

Reply:
In general, I agree. But I have seen a few times that we forgot to specify some env. variables during manual debugging, and got very confused why the failed tests are not reproducible or not run at all on team members' machines.

If we are content with using `cpu` during testing, I think we can keep the current version, i.e. just
```
if "pytest" in sys.modules:
    DEVICE = "cpu"
```
so we can avoid the confusion (especially there are more and more env. variables introduced).

We can change the logic in the future if we find the usage of an env. variable is necessary or preferred.

Being said that, let's see if @sgugger has some opinion.

Reply:
Alright, for me that's good then!